podcasts/Profile - SwiftUI/View Models/ProfileDataViewModel.swift

```diff
@@ -0,0 +1,118 @@
+import Foundation
+import Combine
+import PocketCastsServer
+import PocketCastsDataModel
+
+/// Represents a view that will display information about the users profile such as email, subscription status, and stats
+class ProfileDataViewModel: ObservableObject {
+
+    // Allow UIKit to update to view size changes
+    private(set) var contentSize: CGSize? = nil
+    var viewContentSizeChanged: (() -> Void)? = nil
```

Comment:
Nipticky:

I understand it's a compromise solution to bridge back to UIKit, but I'm not 100% sure this belongs in here?
Feels more like it should live within `ProfileHeaderView` itself perhaps.

Reply:
@jleandroperez I originally had it in the view itself but I moved it here because my thinking was:

1. The view having a direct link to UIKit or outside the "views world" felt like it was branching too far out of "I'm a view and I do view things only". I figured the compromise was to have the view inform the view model that its content size changed, and then the view model can then choose what to do it with without the view caring. 

2. The `ProfileDataViewModel` is a parent class that is used in a later PR by the [`AccountHeaderViewModel`](https://github.com/Automattic/pocket-casts-ios/pull/851/files#diff-8a62c10751d829447595e09ba67440095d0463e7066d5d996517294648900196R5) and moving this here allowed it to be reused in a single place vs having to duplicate the code on each view

Wdyt?

Reply:
Ahhhhh I see (2) is a strong argument. Thanks for the extra context @emilylaguna !!!